life-letters/modernized/12-chapter-11.adoc

```diff
@@ -139,7 +151,7 @@ my own particular affection has lost a pleasing object,
 and the circle of my most near connexions in pure friendship is invaded.--Yet,
 my dear friend, let us look on the other side of the question--for whom we are afflicted?
 One now for ever free from it.
-Recollect we his memory'.' It is precious.
+Recollect we his memory. It is precious.
```

Comment:
I would have let this go if it had been the only thing, but I caught another little thing below, so while you're making changes, can you put the `It is precious.` sentence on it's own line too?